go/node/docker.go

```diff
@@ -0,0 +1,152 @@
+package node
+
+import (
+	"fmt"
+
+	"github.com/obscuronet/go-obscuro/go/common/docker"
+)
+
+type DockerNode struct {
+	cfg *Config
+}
+
+func NewDockerNode(cfg *Config) (Node, error) {
```

Comment:
Nice, so I see you've made this guy pretty much implements the NodeOperator interface, so should be easy to plug in as a networktest env

Reply:
Yeah, hopefully these all plugin and play nicely :)